gradle/verification-metadata.xml

```diff
@@ -2488,121 +2488,241 @@
             <sha256 value="015d5c229f3cd5c0ebf175c1da08d596d94043362ae9d92637d88848c90537c8" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.lucene" name="lucene-analysis-common" version="9.7.0">
+         <artifact name="lucene-analysis-common-9.7.0.jar">
+            <sha256 value="87abf0221c526d64616d47d2261866620c8f42659c93dbfd0d1ffb138af98137" origin="Generated by Gradle"/>
+         </artifact>
```

Comment:
We should remove the old entries, unfortunately the command don't do it for us and we should prevent the file to become huge

Reply:
than you so much Adrien!